server/services/event/hooks/addOpenStatus.js

```diff
@@ -0,0 +1,10 @@
+module.exports = () => (hook) => {
+  const startDate = new Date(hook.result.registrationStartDate);
+  const now = new Date();
+  const endDate = new Date(hook.result.registrationEndDate);
```

Comment:
We can either add timezones to all new `Date`s or to none of them (as done here)